### PR TITLE
feat: extend integration test actions and scheduled tests with optional adminPassword

### PR DESCRIPTION
**Pull request by staticvoid255**
This allows the user to choose to either generate an admin password, or to use the default one.
Also adds test scenarios.

Related to #1055

### CODE REVIEW

**Comment by staticvoid255** [comment]
I have test with both default password and generated one and it works

File: .github/workflows/integration.yml [hunk 1]
```diff
@@ -214,16 +223,25 @@ jobs:
...
-          $domainSuffix
+
+          install_args="--wait --wait-for-jobs --timeout 90m0s otomi chart/otomi \
+            --values tests/integration/${{ inputs.install_profile }}.yaml \
+            --values values-container-registry.yaml
+            --values values.yaml \
+            $domainSuffix"
+
+          if [[ '${{ inputs.generate_password }}' ]]; then
+            password=$(openssl rand -base64 16 | tr -dc A-Za-z0-9)
```
Comment (j-zimnowoda) [inline .github/workflows/integration.yml:238]: Otomi will take care for creating the adminPassword if not provided.

File: .github/workflows/integration.yml [hunk 2]
```diff
@@ -75,6 +79,10 @@ on:
           - no_kms
           - az_kms
         default: az_kms
+      generate_password:
```
Comment (j-zimnowoda) [inline .github/workflows/integration.yml:82]: Could you also make it dropdown ? I think dropdown looks better

**Review by j-zimnowoda — CHANGES_REQUESTED** [review]
Great work functionality as expected. 
I have just few remarks.

**Comment by staticvoid255** [comment]
tested and working, ready for review again

**Review by j-zimnowoda — APPROVED** [review]
Excellent!

**Pull request merged**